meta-buildpacks/nodejs-function/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added `heroku/nodejs-npm-engine` ([#686](https://github.com/heroku/buildpacks-nodejs/pull/686))
+- Added `heroku/nodejs-npm-install` ([#686](https://github.com/heroku/buildpacks-nodejs/pull/686))
+
+### Removed
+
+- Removed `heroku/nodejs-npm` ([#686](https://github.com/heroku/buildpacks-nodejs/pull/686))
+
```

Comment:
Equivalent changelog entries for the non-functions composite buildpack (for the changes made in #623, #625 and #685) are missing from `meta-buildpacks/nodejs/CHANGELOG.md`. Could you add them before the next CNB release? (Either in a separate PR or here, whatever you prefer).

Reply:
Fixed in https://github.com/heroku/buildpacks-nodejs/pull/697